sdk/eventhubs/microsoft-azure-eventhubs/pom.xml

```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-client-authentication</artifactId>
-      <version>1.6.15</version> <!-- {x-version-update;com.microsoft.azure:azure-client-authentication;external_dependency} -->
+      <version>1.7.2</version> <!-- {x-version-update;com.microsoft.azure:azure-client-authentication;external_dependency} -->
```

Comment:
Should this have the relocation section added since there is a Track 2 client for EventHubs?

Reply:
That is a good point Alan - ideally yes, we should start relocating as soon as we believe we have a replacement for the existing track one library.